src/backend/marsha/bbb/urls.py

```diff
@@ -12,12 +12,18 @@
 
 router = DefaultRouter()
 router.register("classrooms", ClassroomViewSet, basename="classrooms")
+router.register("meetings", ClassroomViewSet, basename="meetings")
 
 urlpatterns = [
     path(
         "lti/classrooms/<uuid:uuid>",
         ClassroomLTIView.as_view(),
         name="classroom_lti_view",
     ),
+    path(
+        "lti/meetings/<uuid:uuid>",
+        ClassroomLTIView.as_view(),
+        name="classroom_lti_view",
```

Comment:
I'm not sure it is "ok" for Django to have the same name for different path, as the new code may not rely on this one, maybe you can use another name to assert it is never used.

Reply:
I forgot to change it.
It appears that is no problem for django, but if you call a reverse route for 'classroom_lti_view', you'll get the second one : 'lti/meetings/…' which it was not intended in our case.